apps/package.json

```diff
@@ -184,8 +184,8 @@
     "scratch-vm": "v0.1.0-prerelease.1501105826-prerelease.1501105849",
     "script-loader": "^0.7.0",
     "seedrandom": "2.4.2",
-    "sinon": "^2.1.0",
-    "sinon-chai": "^2.9.0",
+    "sinon": "3",
+    "sinon-chai": "2.13",
```

Comment:
You're right - my bad here.  I should be specifying `^2.13.0`, equivalent to `>=2.13.0 <3.0.0`.  I guess there's no reason I shouldn't also write `^3.0.0` for sinon.